modules/ROOT/pages/promote-app-prod-env-design-center.adoc

```diff
@@ -9,29 +9,29 @@ endif::[]
 
 All Mule applications that you create with Flow Designer are created in the Design environment. When a Mule application that you've created is ready to go live, you can deploy it to another environment, if your organization allows you to do so. The target environment must have enough vCores to accommodate the application; otherwise, deployment will fail.
 
-As you deploy your application, you can change connector configurations and reconnection strategies. You need to do this if there are differences between the Design and target environments that would necessitate such changes.
+As you deploy your application, you can change connector configurations and connections. You need to do this if there are differences between the testing and target environments that would necessitate such changes. For example, if you tested your app in the *Sandbox* environment and now want to deploy it to a production environment, you most likely need to replace the connections that each connector used for connecting to test systems with connections for connecting to production systems.
```

Comment:
Are you sure you do not mean "tested your app in the *Design* environment"?
Maybe you are referring to 2 different "Deploy Application" instances, one to Sandbox and the other to Production.

Reply:
Yes, testing in the Design environment be clicking the Test button in FD is one way of testing. Another is deploying to Sandbox. I'm referring to the second method of testing.